measurements/measure.cpp

```diff
@@ -0,0 +1,335 @@
+#include <atomic>
+#include <cassert>
+#include <iostream>
+#include <mutex>
+#include <numeric>
+#include <tests/rcu_ptr_under_test.hpp>
+#include <thread>
+#include <vector>
+
+#include <tbb/queuing_rw_mutex.h>
+#include <tbb/spin_rw_mutex.h>
+
+#include <urcu-pointer.h>
+#include <urcu.h>
+
+class XRcuPtr {
```

Comment:
Why the `X` prefix?

Reply:
X was the name of the original class in the running example. Perhaps we could have `Client` as prefix, or no prefix at all?